### PR TITLE
Update for branding copy changes

### PR DESCRIPTION
**Pull request by galen-mcandrew**
Switching from FIL+ to Fil+, miner to storage provider (aside from certain instances, where there are more dependencies like the Miner Registry)

### CODE REVIEW

**Comment by galen-mcandrew** [comment]
@dkkapur Found some legacy FIL+, and also might as well start updating to "storage provider"

**Review by dkkapur — APPROVED** [review]
Thanks for this!

File: docs/store/filecoin-plus.md [hunk 1]
```diff
@@ -102,13 +102,13 @@ _Note: [Lotus-shed](https://github.com/filecoin-project/lotus/tree/master/cmd/lo
 lotus-shed verifreg check-client f00000
 ```
 
-### Finding miners to take FIL+ deals
+### Finding storage providers to take Fil+ deals
```
Comment (johnnymatthews) [inline docs/store/filecoin-plus.md:105]: This is a tricky one. _FIL_ is shorthand for _filecoin_, which is the cryptocurrency of the _Filecoin_ network. I wouldn't use _FIL_ to represent the _Filecoin Plus_ program. Saying that, if the program is referred to as _Fil+_ in other places, then that's fine. Not ideal, but it's not terrible.

**Review by johnnymatthews — APPROVED** [review]

File: docs/store/filecoin-plus.md (hunk 1 above)
Comment (dkkapur) [reply docs/store/filecoin-plus.md:105]: Right - this is why we are switching from FIL to Fil everywhere, so at least the capitalization is different / slightly less misleading. Agree that maybe we need to switch to something even more different, but thats a longer term conversation.

Comment (johnnymatthews) [reply docs/store/filecoin-plus.md:105]: Sounds good.

**Pull request merged**